test/TestAssets/performance/NUnit1Passing/NunitTests.tt

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace NUnitAdapterPerfTests
+{
+
+	public class NunitTests
```

Comment:
There is a mix of spaces and tabs, could you save the file from VS to have it fixed please?